authors/settings.py

```diff
@@ -122,6 +122,10 @@
 
 STATIC_URL = '/static/'
 
+DJANGO_SETTINGS_MODULE = [
```

Comment:
@engjames This seems unnecessary. Any purpose for it?

Reply:
Alright am going to remove it I was getting an error of it improperly set. But all now is well and thanks for the observation.